src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -2,17 +2,43 @@
 
 public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-    int totalRejected = 0;
+    private int totalRejected = 0;
 
+
+    /**
+     *
+     * @return zwraca totalreject
+     */
     public int reportRejected() {
-        return totalRejected;
+        return getTotalRejected();
     }
 
+    /**
+     *
+     * @param in przyjmuje liczbe jaka ma byc dodana do listy
+     */
+    @Override
     public void countIn(int in) {
         if (!callCheck() && in > peekaboo()) {
-            totalRejected++;
+            setTotalRejected(getTotalRejected() + 1);
         } else {
             super.countIn(in);
         }
     }
+
+    /**
+     *
+     * @return zwraca totalreject
+     */
+    public int getTotalRejected() {

```

Comment:
Shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internally usually do not require encapsulation. 
Only reportRejected() is part of API